### DISCUSSION TITLE
Using only distributed cache

### DISCUSSION THREAD

**Discussion by efortunov** [ZiggyCreatures/FusionCache#129] [Q&A] [answered]
Hello. How can I use only distributed cache without memory cache?

**Comment by jodydonetti** [comment, marked as answer]
Hi @efortunov , currently it's not possible.
See [here](https://github.com/ZiggyCreatures/FusionCache/discussions/59) for other community members asking for the same thing, and [here](https://github.com/ZiggyCreatures/FusionCache/discussions/59#discussioncomment-3224499) for a detailed response.

Hope this helps.
